src/pages/ProfilePage.js

```diff
@@ -102,6 +106,7 @@ function ProfilePage(props) {
     }, [accountID]);
 
     const details = lodashGet(props.personalDetails, accountID, {isLoading: ValidationUtils.isValidAccountRoute(accountID)});
+    const isExistingReport = ReportUtils.getChatByParticipantInclude(accountID);
```

Comment:
Does this mean that you check the personal details, chat with the user or not, and decide to show the information based on that?

Can we remove that personal detail when the user is removed from the workspace instead of checking if the account has chatted with the user?

Reply:
If it's possible.If we do that, we don't need to check with the user with the existing chat, right?

Reply:
I mean not in the admin account, but instead on the second account that receives pusher notification on removed workspace members.

Reply:
> Yes, the second account received a pusher notification on the removed workspace members. But how can we do something when received this notification? 

That's what I thought; that's why I'm asking. Maybe you know how to do that 😅

But thanks @dukenv0307 for the update; I'll start testing it.